containers_test.go

```diff
@@ -0,0 +1,23 @@
+package proxmox
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStartContainer(t *testing.T) {
+	client := ClientFromLogins()
+	node, err := client.Node("")
```

Comment:
might need to pull Nodes and maybe randomly select a node in the cluster to run this on